doc/u-boot-env-salvator-x-h3-m3.txt

```diff
@@ -0,0 +1,25 @@
+Salvator-X-H3
+===================================
+
+setenv bootdelay 3
+setenv baudrate 115200
+
+setenv bootcmd run bootcmd_xen_tftp
+setenv bootcmd_xen_tftp 'run xen_load_tftp; run dtb_load_tftp; run kernel_load_tftp; run xenpolicy_load_tftp; run initramfs_load_tftp; bootm 0x48080000 0x76000000 0x48000000'
+
+setenv fileaddr 7a000000
+setenv filesize da4e00
+
+setenv dtb_load_tftp tftp 0x48000000 dom0.dtb
+setenv xen_load_tftp tftp 0x48080000 xen-uImage
+setenv initramfs_load_tftp=tftp 0x76000000 uInitramfs
+setenv kernel_load_tftp tftp 0x7a000000 Image
+setenv xenpolicy_load_tftp tftp 0x7c000000 xenpolicy
+
+setenv ethact ravb
+setenv ethaddr 2E:09:0A:00:A0:41
```

Comment:
Whoa! You don't want to do this. Really. There is a sticker with MAC address on every board.

Reply:
it is just an example of what needs to be done on the board, kinda remined

Reply:
done